### DISCUSSION TITLE
Error while updating > How to do manually

### DISCUSSION THREAD

**Discussion by TravelTrader** [tldr-pages/tldr#13198] [General]
tldr worked under Ubuntu 22.04. But yesterday I got no results. Today I updated and it gave a „fatal error: too early EOF“ and I should clone the git manually.

I cloned it in the .config/tldr/ folder but it still doesn’t work.

How can I get the pages work manually?

**Comment by kbdharun** [comment]
Hi, how did you install `tldr` on your device? (we have a lot of clients, so knowing which client you are using will allow us to better assist you).

---

We don't have a lot of Git-based clients, so I am pretty sure you might be using our Python client. Can you try removing the clone and updating the cache manually with `tldr -u`.

**Comment by TravelTrader** [comment]
I installed it with apt.

The `tldr -u` is the one that didn’t work and gave the error. But the clone didn’t work either.

How does the directory `.config/tldr/` have to look like? What folders have to be there exactly please?

**Reply by kbdharun** [reply]
> I installed it with apt.

Hi, unfortunately, I can't help you debug this issue as we don't have an official client packaged in Apt repos , seems like you are using the outdated `tldr-hs` client in Ubuntu 22.04 (this is no longer available in the Ubuntu LTS repos under `tldr` name since 24.04).

I would suggest using our [official Snap client instead](https://snapcraft.io/tldr) which runs our Python client and is publically packaged and maintained [here](https://github.com/tldr-pages/tldr-python-client/blob/main/snap/snapcraft.yaml) since you are on Ubuntu.

**You can get it with the command `snap install tldr`.**

> How does the directory .config/tldr/ have to look like? What folders have to be there exactly please?

It depends on the client, my client uses the XDG path and stores the extracted pages over there.

**Reply by leomilano** [reply]
Thanks, @kbdharun - I was having the error below, and installing the snap package (after removing the deb package)  fixed the issue. 

Downloading tldr pages to /home/lmilano/.local/share/tldr 
tldr: Data.Binary.Get.runGet at position 4: Did not find end of central directory signature
